composer.json

```diff
@@ -25,7 +25,14 @@
     },
     "autoload": {
         "psr-4": {
-            "com\\boxalino\\bxclient\\v1\\": "lib"
+            "com\\boxalino\\bxclient\\v1\\": "lib",
+            "Thrift\\Transport\\": "lib/Thrift/Transport"
+        },
+        "classmap": ["lib/Types.php", "lib/P13nService.php"]
```

Comment:
since these files are auto generated and contain multiple classes, we have to use classmap here to autoload those classes